apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QIntegrationTestBase.java

```diff
@@ -247,6 +230,23 @@ protected Map<String, Object> map(
     return map;
   }
 
+  protected Map<String, Object> map(
+      String key1,
+      Object value1,
+      String key2,
+      Object value2,
+      String key3,
+      Object value3,
+      String key4,
+      Object value4) {
+    Map<String, Object> map = new LinkedHashMap<>();
```

Comment:
Since we use JDK 17, could also use `Map.of(....)`

Reply:
Removed the new method and used `Map.of`